js/index.js

```diff
@@ -1,42 +1,73 @@
 // ITERATION 1
 
 function updateSubtotal(product) {
-  console.log('Calculating subtotal, yey!');
+  const price = product.querySelector('.price span').innerHTML;
```

Comment:
```suggestion
  const price = product.querySelector('.price span').innerText;
```
Siempre que puedas evitarte usar HTML mejor :D